### ISSUE TITLE
Is it possible to make it multiplayer?

### ISSUE THREAD

**Issue by LarryJane491** [db0/godot-card-game-framework#185]
Hello! Thank you so much for this template, this is EXACTLY what I was looking for! This makes building a card game videogame so easy! I've already integrated my cards in the project.

Now I only have one issue: the multiplayer button doesn't seem to do anything. So I'm wondering: is it possible to make a multiplayer game?
I'm brand new to Godot, but I intend to learn a bit more (just enough to make my card game x) ). If I have to check out another source to integrate multiplayer, I'm willing to do so ^^. But since you already added a multiplayer button, maybe you already have a solution for that?

Thank you very much!

**Comment by db0** [comment]
yes it's possible. I already did some attempts at it in the `nakama` branch, but I didn't have time to work on it further. But it's definitelly possible.

**Comment by LarryJane491** [comment]
No problem, thank you for the answer! Do you know a good resource to learn how to make multiplayer in Godot? I'll check it out myself too, but if you already have a good resource, i'll take it ^^.

**Comment by LarryJane491** [comment]
Ah yes, I noticed this video, thanks!
